target/linux/ar71xx/base-files/etc/diag.sh

```diff
@@ -30,6 +30,9 @@ get_status_led() {
 	xd3200)
 		status_led="$board:green:system"
 		;;
+	ap531b0)
```

Comment:
Dear pepe2k,
i`m little confused about alphabetical order.
should i put my code after " antminer-r1|\  " ?